zcash_transparent/src/coinbase.rs

```diff
@@ -0,0 +1,108 @@
+//! Types, constants, and functions related to coinbase transactions.
+
+use alloc::{fmt, vec::Vec};
+use zcash_script::{opcode::Evaluable, pv::push_value};
+
+/// The maximum length of the coinbase transaction script.
+pub const MAX_COINBASE_SCRIPT_LEN: usize = 100;
+
+/// The minimum length of the coinbase transaction script.
+pub const MIN_COINBASE_SCRIPT_LEN: usize = 2;
```

Comment:
I confirmed that these match the consensus rules implemented in `zcashd`:
https://github.com/zcash/zcash/blob/16ac743764a513e41dafb2cd79c2417c5bb41e81/src/main.cpp#L1754-L1755